src/shim/fetch.ts

```diff
@@ -1,5 +1,38 @@
-import global from './global';
-`!has('build-elide')`;
-import 'whatwg-fetch';
+import dojoGlobal from './global';
+import has from '../has/has';
 
-export default global.fetch.bind(global) as (input: RequestInfo, init?: RequestInit) => Promise<Response>;
+let fetchLoadedPromise: Promise<void>;
+if (!has('build-elide')) {
```

Comment:
I'm not sure what the best way to resolve this would be then. Seems like what we'd really want is two separate static imports, one with 
`!has('build-elide')` as well as `has('host-node')` and one for the browser. 

But I'm not sure that would behave as expected.

Reply:
so I think we don't actually ever need to run fetch in node as node is only a supported target environment under test. given thats the complaint (whatwg-fetch blowing up in tests), we can perhaps just use a polyfill that doesn't, and then we don't need to complicate this. For example: https://github.com/developit/unfetch I think doesn't blow up.